aten/src/ATen/native/native_functions.yaml

```diff
@@ -145,6 +145,7 @@
 
 - func: rename_(Tensor(a!) self, Dimname[]? names) -> Tensor(a!)
   variants: method
+  tags: inplace_view
```

Comment:
this doesn't impact anything since it's composite, but technically this op is an inplace_view so I added it here